laws.md

```diff
@@ -83,4 +81,6 @@ It is important, for parsing, that no other non-space characters be included in
 
 **319.** VOTING INCENTIVE: All players who vote on rule-change proposal shall roll 2d6+1 and receive a number of points equal to the value of the roll if and only if the proposal reaches a quorum.
 
-**320.** No point values may be manipulated outside of the rules. 
+**320.** No point values may be manipulated outside of the rules.
+
+**326.** A player may only become active upon submitting a rule in any form or a majority vote of the then active players. Given a tie vote the player is to be immediately added as active in the spirit of furthering the game. Also, let stand, that no individual may control more than one active player.
```

Comment:
The wording of this rule is confusing to me. Does the submission of a rule cause a player to become active? Is that what this says? I don't understand why it says "or a majority vote of the then active players." A majority vote on what? Adding the player as active or the rule the player proposed? You're also going to want to better define a tie vote. If no one has voted is it a tie vote? If I vote yay and another active player votes nay, is that a tie vote? It hasn't reached quorum yet. Even if it has reached quorum and it's a tie another active player could vote and it would no longer be a tie.